File: JavaScript/1/1_1.js
function MathEquation(a, b) {
  let equation = Math.sqrt(a * a + b * b);
  console.log("The result is: " + equation);
  return equation;
}

let x = 3;
let y = 4;
let result = MathEquation(x, y);
console.log("Computed value: " + result);

File: C#/1/1_1.cs
using System;

class Program
{
    static void Main()
    {
        double a = 3;
        double b = 4;
        double c = c(a, b);
        Console.WriteLine("Computed value: " + c);
    }

    static double pythagorasSentence(double a, double b)
    {
        double c = Math.Sqrt(a * a + b * b);
        Console.WriteLine("The result is: " + c);
        return c;
    }
}
